src/Components/WorkloadRules/WorkloadRules.js

```diff
@@ -198,6 +227,17 @@ const WorkloadRules = ({ workload }) => {
     },
   };
 
+  const onSort = (_e, index, direction) => {
+    setRowsFiltered(false);
+    setExpandFirst(false);
+    setFirstRule('');
+    return updateFilters({
```

Comment:
`updateFilters` doesn't return anything. Let's just call the function.

```suggestion
    updateFilters({
```

Reply:
It's not removed in the last commit

Reply:
That's a mistake on my end, forgot to add this change while I was jumping between 2 branches. I am pushing new update right now